src/components/state-details.js

```diff
@@ -355,7 +369,11 @@ export default function StateDetailsPage ({ location, data }) {
                 </p>
 
                 <p className="mt-5">
-                  To cut this pollution, replace your car with an EV.
+                  To cut this pollution, if you have a car, your next one needs to be an electric vehicle (EV).
+                </p>
+
+                <p className="mt-5 mb-0">
+                  <img className="img-fluid" src={CarTransition} title="Your next car needs to be an electric vehicle (EV)" alt="Your next car needs to be an electric vehicle (EV)" />
```

Comment:
Same question about `title`. Also this `alt` doesn't seem appropriate - if this image is _decorative_ (it doesn't serve a content purpose) it can have an empty alt (as in `alt=""`) but otherwise I think an alt like "Gas emitting car being converted to electric car". See [https://webaim.org/techniques/alttext/](Web AIM's article on alt text).

Reply:
good point. I hadn't thought of this and the context in which `alt` is used. I'll update this to be more descriptive